apps/user-portal/src/components/account-recovery/options/email-recovery.tsx

```diff
@@ -117,11 +103,11 @@ export const EmailRecovery: React.FunctionComponent<EmailRecoveryProps> = (
      */
     const setEmailAddress = (response) => {
         let emailAddress = "";
-        response.emails
-            ? response.emails.map((email) => {
-                emailAddress = email;
-            })
-            : null;
+        if (response.emails) {
+            response.emails.map((emailParam) => {
+                emailAddress = emailParam;
```

Comment:
If we are getting a an array of email addresses in the response shouldn't we also use an array here?